### PR TITLE
[feature] Added Pipenv support

### PR DESCRIPTION
**Pull request by SergeyKons**
Added Pipenv support
There are small problem with multiple empty lines on Pipenv file

Read about pipenv pipenv.readthedocs.io

### CODE REVIEW

**Comment by SergeyKons** [comment]
Fixed test_start.py because of new file in template dir

**Comment by SergeyKons** [comment]
Forgot fix test_start_simple in ea07007
Typos ("Pipev" "Pipfile") in 1b12606

**Comment by SergeyKons** [comment]
test_start_simple: 18 > 19 files

File: README.rst [hunk 1]
```diff
@@ -142,6 +142,12 @@ Before you can run your app you'll need to install the other requirements, lucki
 
     pip install -r requirements.txt
 
+Or you can use pipenv (if it is already installed in your env). For install packages and lock it on Pipfile.lock, use next command
```
Comment (webknjaz) [inline README.rst]:
```suggestion
Or, you can use `pipenv`_ (if it is already installed in your env). To install packages and lock it on ``Pipfile.lock``, use next command
```

**Comment by SergeyKons** [comment]
Pipenv used by many projects. Also PyCharm has pipenv support. I'm not looked for poetry yet. Imo it willl be better if user can select beetween requerements.txt, poetry and pipenv

P.s. sry for ban english

**Comment by SergeyKons** [comment]
Hmm... Why travis build failing? All tests passing correctly now

**Comment by webknjaz** [comment]
Linters. Read build logs.

**Comment by SergeyKons** [comment]
Yeah i see it, but this files not edited on this pr

**Comment by samuelcolvin** [comment]
personally I don't use pipenv, why have it as well as `requirements.txt` which is the de facto default?

The build is failing because isort isn't properly pinned and has updated and changed its rules. I'll fix that tomorrow.

**Comment by SergeyKons** [comment]
Thanks. For me its more similar to node's packages.json. I can store all dependencies in one file. And as i can see pipenv going popular

**Comment by webknjaz** [comment]
That "popularity" is artificial. It brings more problems than it solves. https://chriswarrick.com/blog/2018/07/17/pipenv-promises-a-lot-delivers-very-little/

**Comment by samuelcolvin** [comment]
I don't have a view on pipenv, but I think we should keep the app template as simple as possible, that means we should only have one list of packages and I think `requirements.txt` is the most obvious option.

**Pull request closed without merging**